### DISCUSSION TITLE
SIC Internacional [PT]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#371] [Show and tell]
### Channel ID

SICInternacional.pt

### Stream URL

http://live.impresa.pt/live/sicint/sicint.m3u8

### Notes

Geo-blocked

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by AntiPontifex** [comment]
Geo-blocked (from Canada)? Thank you for the stream: it will be included in the next release.

**Comment by BellezaEmporium** [comment]
geo to portugal only

**Comment by freearhey** [comment]
http://live.impresa.pt/live/sicint/sicint.m3u8 (offline)
<img width="307" alt="image" src="https://user-images.githubusercontent.com/7253922/232254120-a55915dd-7896-4055-827d-6b07ca17e574.png">
